compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -750,6 +750,7 @@ class Definitions {
   @tu lazy val RefiningAnnotationClass: ClassSymbol = ctx.requiredClass("scala.annotation.RefiningAnnotation")
 
   // Annotation classes
+  @tu lazy val ScalaAnnotationInternal: ClassSymbol = ctx.requiredClass("scala.annotation.internal")
```

Comment:
Does not seem to be needed, and is weird anyway, since `internal` is a package, not a class.